example.yaml

```diff
@@ -9,6 +9,10 @@ storage:
   host: localhost # redis hostname. not required for memory
   port: 6379      # redis port.     not required for memory
 
+health-check:
```

Comment:
Rather than doing string comparisons throughout with the port, how about having a `enabled: true` bool that we can put into the HealthCheck struct?